staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -693,8 +696,26 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 				return nil, err
 			}
 		}
-		// TODO: Once we get rid of /healthz consider changing this to post-start-hook.
-		err := s.addReadyzChecks(healthz.NewInformerSyncHealthz(c.SharedInformerFactory))
+		err = s.AddPostStartHook("generic-apiserver-informers-synced", func(context PostStartHookContext) error {
```

Comment:
don't we need a check like in https://github.com/openshift/kubernetes/pull/339/files#diff-c15d69437252be511f1f5e1d9affbaafR689 ?

Reply:
yes, we do. thanks.